lib/kb_uploadmethods/kb_uploadmethodsImpl.py

```diff
@@ -176,6 +176,11 @@ def upload_fasta_gff_file(self, ctx, params):
         print('--->\nRunning uploadmethods.upload_fasta_gff_file\nparams:')
         print((json.dumps(params, indent=1)))
 
+        if params.get('ncbi_taxon_id'):
```

Comment:
there is no `ncbi_taxon_id` and `relation_engine_timestamp_ms` UI component defined. So `params.get('ncbi_taxon_id')` will never be `True`. And why are you assigning `ncbi_taxon_id` to `taxon_id`? why not just use the existing `taxon_id`? or rename `taxon_id` to `ncbi_taxon_id`?

Reply:
I've added both ncbi_taxon_id and relation_engine_timestamp_ms to the app input parameter spec in kb_uploadmethods.spec.  'ncbi_taxon_id' is the 'selection_id' by the Relation Engine search, while taxon_id is the parameter passing from kb_uploadmethods apps to GFU.